source/shared/cpp/ObjectModel/BaseActionElement.h

```diff
@@ -38,6 +38,8 @@ class BaseActionElement
     Json::Value m_additionalProperties;
 
 protected:
+    virtual void PopulateKnownPropertiesSet(void) = 0;
```

Comment:
> = 0 [](start = 49, length = 4)

So, the = 0 here is misleading since this class does actually implement this method. If you want to make this pure virtual in order to make sure that subclasses implement it, you can just set the base card properties in the constructor directly.